CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
   - limit for outgoing gRPC messages has changed from 2147483647 to 16777216 bytes
   - limit for incoming gRPC messages has changed from 4194304 to 104857600 bytes
 * [FEATURE] Distributor/Ingester: Provide ability to not overflow writes in the presence of a leaving or unhealthy ingester. This allows for more efficient ingester rolling restarts. #3305
+* [FEATURE] Query-frontend: introduced query statistics logged in the query-frontend when enabled via `-frontend.query-stats-enabled=true`. When enabled the following metrics are also tracked: #3539
+  - `cortex_query_seconds_total`
```

Comment:
Can you explain somewhere (e.g. in the docs) how this metric differers from `cortex_request_duration_seconds{route="api_prom_api_v1_query_range"}`

Reply:
Yes, sure. Done.